inst/extdata/examples/TestProject/Data/Digitized Figures/README.md

```diff
@@ -0,0 +1 @@
+Where to store figures which data was extracted from
```

Comment:
```suggestion
Where to store figures which data was extracted from
```

Folder for storing source figures and WebPlotDigitizer (or similar extraction software) projects used for digitizing data.

Reply:
Commit suggestion and merge :D